aws_okta_keyman/keyman.py

```diff
@@ -109,8 +109,7 @@ def main(self):
     @staticmethod
     def setup_logging():
         """Return back a pretty color-coded logger."""
-        logger = logging.getLogger()
-        logger.setLevel(logging.INFO)
```

Comment:
Removing this line removes most of the logging used in Keyman today which is primarily using INFO. This can be avoided and allow both cases with something like the below code. Not sure I like it but it would do the thing. 

        if __name__ == 'aws_okta_keyman.keyman':
            logger.setLevel(logging.INFO)

Reply:
True, if you don't get and set the level externally. Perhaps instead, the logging should be controlled in your `main.py`. In `keymain.py`, you simply have `LOG = logging.getLogger(__name__)`. In `main.py`, you setup the colorlog handler and set level to INFO. This would similar to the design in `aws.py` for example.

Reply:
That's true. I'll play with this a bit locally and see what I like. I'm definitely in support of the idea of making it simpler to use as a lib if that's what someone would prefer to do. My only note would be that at this point I definitely consider the whole thing beta still and may end up moving some methods around as things are refactored and improved.

The biggest changes I'm planning on in that front are in the handling of MFA which could be much more concise.

Reply:
Cool, I'll close the PR. I'm actually using it as a lib, mainly to limit some features (for example, keyring). I appreciate the project, happily stumbled upon it after being unsatisfied with other options. Thanks again!